data/drak/drak ships.txt

```diff
@@ -115,6 +115,27 @@ ship "Archon" "Archon (Cloaked)"
 		"cloak" .04
 	"final explode" "archon c teleport"
 
+ship "Archon" "Archon (Hyperdrive Cloaked)"
+	sprite "ship/archon b"
+		"frame rate" 0.87
+	"never disabled"
+	add attributes
+		"cloak" .04
+		"jump speed" 1
+		"hyperdrive" 1
```

Comment:
```suggestion
ship "Archon" "Archon (Hyperdrive Cloaked)"
	sprite "ship/archon b"
		"frame rate" 0.87
	"never disabled"
	add attributes
		"cloak" .04
		"jump speed" 1
		"hyperdrive" 1
		"hyperdrive sound"
		"hyperdrive out sound"
		"hyperdrive in sound"
```
Adding these to the definition, without defining the sound, makes it silent. This way the player doesn't hear the Archon jumping in every time they move systems. It feels more omnipresent and mysterious this way.